app/src/main/java/org/sopt/and/feature/signin/SignInActivity.kt

```diff
@@ -0,0 +1,94 @@
+package org.sopt.and.feature.signin
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+import org.sopt.and.R
+import org.sopt.and.core.designsystem.component.topappbar.BackButtonTopAppBar
+import org.sopt.and.core.designsystem.theme.ANDANDROIDTheme
+import org.sopt.and.feature.mypage.MyPageActivity
+import org.sopt.and.feature.signup.SignUpActivity
+
+const val ID_KEY = "id"
+const val PASSWORD_KEY = "password"
+
+@AndroidEntryPoint
+class SignInActivity : ComponentActivity() {
+    private var userId: String = ""
+    private var userPassword: String = ""
+
+    val resultLauncher =
+        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+            if (result.resultCode == RESULT_OK) {
+                userId = result.data?.getStringExtra(ID_KEY) ?: ""
```

Comment:
```suggestion
                userId = result.data?.getStringExtra(ID_KEY).orEmpty()
```

이렇게도 쓸 수 있습니다. 그런데 이건 넘모나 취향차이